go/oasis-test-runner/scenario/e2e/runtime/governance_change_parameters.go

```diff
@@ -0,0 +1,136 @@
+package runtime
+
+import (
+	"fmt"
+
+	"github.com/oasisprotocol/oasis-core/go/common/cbor"
+	consensus "github.com/oasisprotocol/oasis-core/go/consensus/api"
+	governance "github.com/oasisprotocol/oasis-core/go/governance/api"
+	"github.com/oasisprotocol/oasis-core/go/oasis-test-runner/env"
+	"github.com/oasisprotocol/oasis-core/go/oasis-test-runner/log"
+	"github.com/oasisprotocol/oasis-core/go/oasis-test-runner/oasis"
+	"github.com/oasisprotocol/oasis-core/go/oasis-test-runner/scenario"
+	"github.com/oasisprotocol/oasis-core/go/oasis-test-runner/scenario/e2e"
+	staking "github.com/oasisprotocol/oasis-core/go/staking/api"
+)
+
```

Comment:
There is probably no need to spin up runtimes when testing these parameter change proposals as they don't really affect them (different to governance proposal for network upgrades)? Actually could we even test these during the "node unit test" given that it is really simple? I want to avoid adding too many e2e tests as each one takes a lot of time to set up the network, especially the runtime ones.

Reply:
Agree. Added unit tests for message handlers and extended node unit test.